components/script_layout_interface/reporter.rs

```diff
@@ -21,7 +23,7 @@ pub struct CSSErrorReporter {
 }
 
 impl ParseErrorReporter for CSSErrorReporter {
-     fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str) {
+     fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str, servo_url : Option<&ServoUrl>) {
```

Comment:
This doesn't seem to be used. We should use it in the log statement (just like we did below).